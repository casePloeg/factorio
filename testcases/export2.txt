run_script "/Users/caseploeg/factorio/factorio-cli/scripts/startup.txt" > /dev/null
set debug true
alias create lprod prod "|" less
alias create prodl lprod
alias create raw prod "|" grep -E "(ore|plate)"
alias create init @ scripts/factory.txt
alias create export history -s ">" export.txt
mine iron-ore 10
place burner-mining-drill iron-ore
place stone-furnace iron-plate
next
