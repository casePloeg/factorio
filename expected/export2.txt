{"assemblers": {}, "current_items": {"burner-mining-drill": 0, "iron-ore": 7, "iron-plate": 23, "stone-furnace": 0, "water": 1000000000}, "current_recipes": ["automation-science-pack", "boiler", "burner-inserter", "burner-mining-drill", "copper-cable", "copper-plate", "electric-mining-drill", "electronic-circuit", "firearm-magazine", "inserter", "iron-chest", "iron-gear-wheel", "iron-plate", "iron-stick", "lab", "light-armor", "offshore-pump", "pipe", "pipe-to-ground", "pistol", "radar", "repair-pack", "small-electric-pole", "steam-engine", "stone-brick", "stone-furnace", "transport-belt", "wooden-chest"], "current_tech": [], "furnaces": {"iron-plate:stone-furnace": 1}, "game_time": 70, "limited_items": {}, "miners": {"iron-ore:burner-mining-drill": 1}}